### PR TITLE
Fix grpc port in examples

### PR DESCRIPTION
**Pull request by captnz**
gRPC port 10001 removed (gRPC now only over 443 like rest of protocols)

### CODE REVIEW

File: examples/identity_auth.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def __init__(
     ):
         split_url = space.partition('://')
         space = split_url[2] or split_url[0]
-        self.grpc_url = space + ':10001'
+        self.grpc_url = space
```
Comment (Evolter) [inline examples/identity_auth.py:34]: I assume it was tested? Won't it be complaining about missing 443 or "s" in protocol.

**Review by Evolter — APPROVED** [review]
LGTM just one comment

**Review by vtermanis — APPROVED** [review]
LGTM

File: examples/identity_auth.py (hunk 1 above)
Comment (vtermanis) [reply examples/identity_auth.py:34]: As discussed internally, this has indeed been tested.

**Pull request merged**